README.md

```diff
@@ -4,364 +4,414 @@ The purpose of the ODM is to support wastewater-based surveillance and epidemiol
 
 The ODM supports data sharing in two ways:
 
-1. **Data sharing schema** - The ODM will have a schema that describes what data can be shared with one or more partners or users.
-2. **Data filter based on the data sharing schema** - The ODM will support an open source method for filtering data tables using the data sharing schema.
+1.  **Data sharing schema** - The ODM will have a schema that describes what data can be shared with one or more partners or users.
+2.  **Data filter based on the data sharing schema** - The ODM will support an open source method for filtering data tables using the data sharing schema.
 
-The data sharing schema will be a csv file (sharing.csv) where each row in the file corresponds to one variable in a ODM table. Attributes in the row describe who the data is shared with and what data is excluded. See below for an example.
+The data sharing schema will be a csv file (sharing.csv) where each row in the `sharing` file corresponds to a header or table in the PHES-ODM. Attributes in the row describe who the data is shared with and what data is included. See below for an example.
```

Comment:
We need a configuration file for a sharing rule. That configuration file would contain metadata for the sharing.csv file, such as:
- the name of sharing schema.
- the database that the sharing schema is for.
- the version, and any notes about updates for the version. 
- contact information for the sharing file. Who created it. 
- date created.
- who created it.
- valid from and to date.

Reply:
I'm wondering if we could talk about this more - it seems to me maybe a bit tedious to have to maintain two files for sharing (or more).

I tentatively added the date created, version, and the valid from and to date (along with status, firstReleased, lastUpdated, and changes). 

I think who created it (datasetID or organizationID), the name of sharing schema, the database that the sharing schema is for (datasetID), any notes about updates for the version, contact information for the sharing file (contactID), and who created it (organizationID or contactID) could also be added as columns in the sharing.csv, but that also makes the table pretty wide. I think two sharing tables might be a lot, but maybe you're right and it's for the best?

Reply:
Addressed it my most recent commit (commit [b35b1fc](https://github.com/Big-Life-Lab/PHES-ODM-sharing/pull/6/commits/b35b1fca29e450ad02a0f0a41612901e24780227))